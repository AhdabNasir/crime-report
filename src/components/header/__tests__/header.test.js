import React from 'react';
import renderer from 'react-test-renderer';
import Header from '..';

describe('<Header/>', () => {
  it('renders <Header/> component', () => {
    const tree = renderer
      .create(<Header text="Page header" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
