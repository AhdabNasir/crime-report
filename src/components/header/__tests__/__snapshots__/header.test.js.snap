// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders <Header/> component 1`] = `
<h1
  className="govuk-heading-l"
>
  Page header
</h1>
`;
